### PR TITLE
added hw

### PR DESCRIPTION
**Pull request by sunniiiw**

### CODE REVIEW

**Comment by chuxu200328** [comment]
image
When mouse hover on the navbar, it should change to a pointer

File: week7/sunny/netflix/src/App.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+body {
+	margin: 0;
+	font-family: 'Helvetica Neue';
```
Comment (chuxu200328) [inline week7/sunny/netflix/src/App.css]: Recommend add two more fonts for backup.

File: week7/sunny/netflix/src/components/Banner/index.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
+.background {
+	background-image: url(../../assets/images/hero.jpg);
+	background-size: cover;
+	width: 100%;
+	margin-top: 88px;
+}
+
+.movie-info-container {
+	color: #fff;
+	padding: 64px 58px 0 58px;
```
Comment (chuxu200328) [inline week7/sunny/netflix/src/components/Banner/index.css:10]: 58px? What's the reason for this?

**Review by chuxu200328 — COMMENTED** [review]
Nice work! But remember:
**DO NOT USE STRANGE NUMBER！！！**

File: week7/sunny/netflix/src/components/Banner/index.css (hunk 2 above)
Comment (sunniiiw) [reply week7/sunny/netflix/src/components/Banner/index.css:10]: i give it a padding instead a margin.

**Review by chuxu200328 — APPROVED** [review]
good job!

**Pull request merged**